ntbs-service/Jobs/UpdateTableCountsJob.cs

```diff
@@ -17,6 +17,8 @@ public class UpdateTableCountsJob
         private readonly ITableCountsRepository _tableCountsRepository;
         private PerformContext _context;
 
+        private const double DecreaseMarginOfError = 0.995; // Allow decreases of less than 0.5%
```

Comment:
I know we agreed 1% for this, but I went back and checked and the only time this job has given us useful information was on 3rd Aug when a change in the way we excluded records caused 12 records to be removed. It wasn't an issue, but it was nice to be reminded about it. In that case it was a decrease of 0.85%. To cover a case like that again, I've gone with setting this to 0.5%. 

This sensitivity, 0.5%, should still stop all but one of the notifications we've had in the last two weeks. The only ones that would get through was when the MBovis tables decrease, because they only have 50-100 records in them, but because there are so few records they don't decrease very often anyway.